src/metrics/include/metrics/operation.hpp

```diff
@@ -387,6 +383,7 @@ class OperationT final {
     explicit OperationT(internals::OperationImpl<ClockSource>& op) : _op{std::addressof(op)} {}
 
     OperationContextT<ClockSource> start() {
+        BOOST_LOG_TRIVIAL(debug) << this->_op->getActorName() << " : " << this->_op->getOpName();
```

Comment:
[Optional] I was using the `Actor : OperationMetric` syntax as an example. I'm not sure if we are trying to keep each log line very succinct on purpose, but I wonder if it would be clearer to write out something a little more verbose like `Started collecting metric {metric} for actor {actor}` just so it's very obvious what the log line means.

Reply:
Oh I actually thought you prefer to keep it succinct. Yes I prefer the more obvious version as well, and have pushed a change for that.

Reply:
I've just taken a look at this again. Turns out adding metric name logging isn't complicated and shouldn't incur any computational cost, so I've added it as well.

Reply:
Turns out my simple change to enable metric name logging actually breaks cmake-test. I've reverted that change, and will investigate why.